packages/block-editor/src/components/inserter/index.native.js

```diff
@@ -234,6 +247,7 @@ export class Inserter extends Component {
 		return (
 			<>
 				{ renderToggle( {
+					canViewEditorOnboarding,
```

Comment:
Not related to the diff here per se but, am I the only one confused with which function `renderToggle()` is? :)

I mean, I see it coming both as a prop and also being a class member. If you are also confused, would it make sense to better differentiate the names as part of this PR? No strong feelings about it.

Reply:
No, I too found myself retracing my steps a few times while deciphering the call stack of `renderToggle`. 😄 Happy to rename one of the instances for clarity.